site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
>So for people who use typescript, what do they distribute to the clients? if they only distribute typescript, then they may need add typescript as a dependency in postinstall

FWIW, typically TS users would include `typescript` as a dev dependency. The npm package would then include the compiled Javascript so that consumers don't have to know or worry about TypeScript. At the end of the day, if the code's on npm it seems like it should be Javascript.